@uportal/waffle-menu/public/index.html

```diff
@@ -31,7 +31,7 @@
   <header>
     <nav>
       <div class="buttons">
-        <waffle-menu debug="true" button-color="red" url="/proxy/uPortal/api/v4-3/dlm/portletRegistry.json" />
+        <waffle-menu debug="true" button-color="red" url="portletRegistry.json" truncate-length="50" />
```

Comment:
overriding the default value, with the default value may not be needed :slightly_smiling_face: